apps/passport-server/.env.example

```diff
@@ -1,70 +1,217 @@
+# node -e 'console.log(require("@pcd/eddsa-pcd").newEdDSAPrivateKey())' | pbcopy
+SERVER_EDDSA_PRIVATE_KEY="860d116de20c1cb5bd7cb68fdb786da9d9fb35bf96c336d6dcaef64733701f20"
+
+# Account resets are strictly rate-limited. However, this can be turned off
+# by setting the following environment variable to 'true'.
+# If GENERIC_RATE_LIMIT_DISABLED='true' then *all* rate-limiting is disabled,
+# irrespective of whether ACCOUNT_RESET_RATE_LIMIT_DISABLED is set.
+ACCOUNT_RESET_RATE_LIMIT_DISABLED='true'
+
+# Set this to an integer value between 1 and the number of
+# CPU cores you have available. The server offloads CPU-bound
+# tasks to these workers
+WORKER_QUANTITY=1
+
+##################################################################
+##################################################################
+#   Frogcrypto
+##################################################################
+##################################################################
+
+# A stringified JSON array of email addresses whose Zupass user can manage FrogCrypto via UI and api routes
+# e.g. '["test@example.com"]'
+# FROGCRYPTO_ADMIN_USER_EMAILS=
+
+# The base URL for frogcrypto assets
+# FROGCRYPTO_ASSETS_URL=
+
+##################################################################
+##################################################################
+#   Podbox.dev (localhost:3002) (client on localhost:3005)
+##################################################################
+##################################################################
+
+# So that Podbox server knows the url of where its corresponding client lives.
+# 
+GENERIC_ISSUANCE_CLIENT_URL="http://localhost:3005"
+
+# To enable server-issued PCDs using the generic issuance feature, the server needs an an EdDSA 
+# private key. You can generate one using the following command at the root of the project, 
+# after installing dependencies and building the project:
+# 
+# node -e 'console.log(require("@pcd/eddsa-pcd").newEdDSAPrivateKey())' | pbcopy
+GENERIC_ISSUANCE_EDDSA_PRIVATE_KEY="129806f9d68c3cf87450a968748fd31f01ba4e6f2078d80f24ebab23d98df7cf"
```

Comment:
double-checking that we're not using this anywhere?

Reply:
yeah i generated all these keys fresh for this PR